util/cluster.go

```diff
@@ -132,15 +144,17 @@ func CreateBackrestRepoSecrets(clientset *kubernetes.Clientset,
 			},
 		},
 		Data: map[string][]byte{
-			"authorized_keys":         keys.Public,
-			"aws-s3-ca.crt":           configs.Data["aws-s3-ca.crt"],
-			"aws-s3-credentials.yaml": s3KeySecretData,
-			"config":                  configs.Data["config"],
-			"id_ed25519":              keys.Private,
-			"sshd_config":             configs.Data["sshd_config"],
-			"ssh_host_ed25519_key":    keys.Private,
+			BackRestRepoSecretKeyAWSS3KeyAWSS3Key:       backrestS3Key,
+			BackRestRepoSecretKeyAWSS3KeyAWSS3KeySecret: backrestS3KeySecret,
+			backRestRepoSecretKeyAuthorizedKeys:         keys.Public,
+			backRestRepoSecretKeyAWSS3KeyAWSS3CACert:    configs.Data["aws-s3-ca.crt"],
```

Comment:
Can we use the `const` defined above here instead of the hardcoded string? E.g.:
`configs.Data[backRestRepoSecretKeyAWSS3KeyAWSS3CACert]`

Same would apply to the next two as well.

Reply:
How'd I miss that... 😐 Yes, I'll fix.